docs/argh_derive_expand/README.md

```diff
@@ -0,0 +1,681 @@
+# How to debug the expanded derive marco for `argh`
+
+The `argh::FromArgs` derive marco can be debugged with the [cargo-expand](https://crates.io/crates/cargo-expand) crate.
+
+This module is an expand on how you could debug such an expansion.
+
+## Expand the derive marco in `src/lib.rs`
+
+See [src/lib.rs](./src/lib.rs) for the example struct we wish to expand.
+
+First, install `cargo-expand` by running `cargo install cargo-expand`.
+
+Once installed, run `cargo expand` on the command line and you can see the expanded code.
+
+## Example
```

Comment:
While I like the idea of documenting how to see what the macro produces, I'm worried that the generated code that you embed in the documentation will get out of date. What do you think about putting the above two sections about suggesting how to use `cargo-expand` into the toplevel README, but leave out the generated code?

Reply:
Done.